tests/instruments/ametek/test_ametek7270.py

```diff
@@ -0,0 +1,508 @@
+import pytest
+
+from pymeasure.test import expected_protocol
+from pymeasure.instruments.ametek import Ametek7270
+
+
+def test_init():
+    with expected_protocol(
+            Ametek7270,
+            [],
+    ):
+        pass  # Verify the expected communication.
+
+
+@pytest.mark.parametrize("comm_pairs, value", (
+    ([(b'SEN', b'27\n')],
+     1.0),
+    ([(b'SEN', b'27\n')],
+     1.0),
```

Comment:
What's the difference between those two cases?

Reply:
well it was autogenerated from your auto code...I'll change that